Sources/MongoSwift/Operations/ListDatabasesOperation.swift

```diff
@@ -41,7 +41,7 @@ internal struct ListDatabasesOperation: Operation {
 
     internal func execute(using connection: Connection, session: ClientSession?) throws -> ListDatabasesResults {
         // spec requires that this command be run against the primary.
-        let readPref = ReadPreference(.primary)
```

Comment:
this command must be explicitly run against the primary, so I think we can leave this as-is.